sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusRetryOptions.cs

```diff
@@ -35,10 +35,8 @@ public class ServiceBusRetryOptions
         ///   The maximum number of retry attempts before considering the associated operation
         ///   to have failed.
         /// </summary>
-        ///
-        ///
         /// <value>The default retry limit is 3.</value>
-        ///
+        /// <exception cref="ArgumentOutOfRangeException">Occurs when the requested retries is not between 0 and 100 (inclusive).</exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentOutOfRangeException">Occurs when the requested number of retries is not between 0 and 100 (inclusive).</exception>
```

Would you mind reflecting that to the EH version too?   I, apparently, don't talk more gooder here.